docs/metrics/accelerate.md

```diff
@@ -9,80 +9,86 @@ Pulse calculates the four key Accelerate metrics using strictly **changes**, **d
 
-How often your organization completes a deployment to production or releases code to end-users of your primary application or service.
+How often your organization completes a deployment to production or releases code to end-users of your primary application or service:
 
-The most specific that applies:
+```text
+number of deployments per day
+```
 
-| Deployment frequency | Value                                                                                                                                  |
-| -------------------- | -------------------------------------------------------------------------------------------------------------------------------------- |
-| On-demand            | Median of more than one `deployments` per day (i.e. more than 50% of the days have more than one deployment.)                          |
-| Daily                | Median of three or more days with `deployments` in a calendar week (i.e. more than 50% of weeks have 3 or more days with deployments.) |
-| Weekly               | Median of one or more `deployments` in a calendar week (i.e. more than 50% of weeks have at least one deployment.)                     |
-| Monthly              | Median of one or more `deployments` in a calendar month (i.e. more than 50% of months have at least one deployment.)                   |
-| Yearly               | None of the above                                                                                                                      |
+Pulse uses the average value over a period of time to display aggregate deployment frequencies.
 
-!!! note
-    The value above is used to determine your Performance level. The numeric value displayed as Deployment Frequency is the number of `deployments` per day.
-
-When we aggregate Deployment Frequency we use the average of a period of time.
+Pulse determines your performance level for this metric as follows:
 
-| Performance level[^1] | Value                                                              |
-| --------------------- | ------------------------------------------------------------------ |
-| Low                   | Monthly or Yearly (Between once per month and once every 6 months) |
-| Medium                | Weekly (Between once per week and once per month)                  |
-| High                  | Daily (Between once per day and once per week)                     |
-| Elite                 | On-demand (Multiple deployments per day)                           |
+| Performance level[^1] | Deployment frequency                                                  |
+| --------------------- | --------------------------------------------------------------------- |
+| Elite                 | Median of more than one **deployment** per day                        |
+| High                  | Median of three or more days with **deployments** in a calendar week) |
+| Medium                | Median of one or more **deployments** in a calendar week              |
```

Comment:
I actually meant that I prefer having:
```suggestion
| Medium                | More than 50% of weeks have at least one deployment             |
```

because to me, the existing sentence drives me to think of the median of deployments, which doesn't make much sense.

Reply:
Aah, I see! I agree it's more clear like this. :+1: